content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -242,6 +242,12 @@ where `config.yaml` contains the custom `imageRepository`, and/or `imageTag`
 for etcd and CoreDNS.
 * Pass the same `config.yaml` to `kubeadm init`.
 
+#### Custom sandbox (pause) images
+
+Container runtimes are responsible for their sandbox images, a common default being `k8s.gcr.io/pause`.
+To set a custom image for these you need to configure this in your [container runtime](/docs/setup/production-environment/container-runtimes/)
+to use the image.
+
```

Comment:
We don't document all container runtimes and we don't have a lot of capacity to maintain docs for
lots of container runtimes. Ideally we'd be able to link to container runtimes' docs on this, however those docs aren't there to link to.

So, we need to compromise.

How about:

```suggestion
To set a custom image for these you need to configure this in your 
{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
to use the image.

Consult the documentation for your container runtime to find out how to change this setting;
for selected container runtimes, you can also find advice within the
[Container Runtimes]((/docs/setup/production-environment/container-runtimes/) topic.
```

Reply:
:thought_balloon: are there any container runtimes that work properly with Kubernetes and that don't use a pause image at all?